docs/mpcie_modules.md

```diff
@@ -162,6 +162,50 @@ operating system.
 
 (1) Refer to quirks, it is known to not work stably on all apu2 board revisions
 
+## Ethernet controllers
+
+### Dual Ethernet Controller Realtek RTL8111
+
+![RTL8111](images/realtek_controller.jpg)
+
+This model works well with apu1 boards. However, it is not seen in apu2 and apu3
+boards with default coreboot settings. To enable it make following steps:
+
+1. On apu2 board make sure to enable option *force mPCIe2 clk* in Payload menu.
```

Comment:
@piotr-kleins please be more precise. The setting is required only when placing the module in mPCIe2 slot. Otherwise, the change does not make any difference. Point out that which port does have problems and which not. Preferably in a form of table like I did before (see other modules)

Reply:
UPDATE: I see You have the table already. Please make a side note `&sup1;` in the cell with apu2 mPCIe2 like I did here https://github.com/pcengines/apu2-documentation/pull/147/files#diff-73e2b4e6830f15d7fe8b1b2cbd1ba882R159
and then explain that force mPCIe2 clock must be enabled

Reply:
In the table I gave a short notification about apu2 mPCIe2 slot